deposit/src/test/java/edu/unc/lib/deposit/transfer/TransferBinariesToStorageJobTest.java

```diff
@@ -366,7 +366,7 @@ public void depositResumeFailed() throws Exception {
         assertFitsFileTransferred(postFileResc2);
 
         verify(jobStatusFactory, times(2)).setTotalCompletion(eq(jobUUID), eq(5));
-        verify(jobStatusFactory, times(4)).incrCompletion(eq(jobUUID), eq(1));
```

Comment:
I think you might have to reset the click count between runs, otherwise the count is going to be higher than the total after a resume. Not sure if trying to skip `addClick` during a resume would be practical, but maybe its possible?